src/components/oscal-utils/OSCALProfileResolver.js

```diff
@@ -79,13 +79,13 @@ export default function OSCALResolveProfileOrCatalogUrlControls(
           inheritedOSCALObject.inherited = [];
 
           modifications["set-parameters"].push(
-            ...(result.profile.modify["set-parameters"] ?? [])
+            ...(result.profile.modify?.["set-parameters"] ?? [])
           );
-          modifications.alters.push(...result.profile.modify.alters);
+          modifications.alters.push(...(result.profile.modify?.alters ?? []));
```

Comment:
I feel like these didn't come from your branch? What's going on here? Weird rebase/merge? These changes are already present on `develop`

Reply:
Should be fixed.